spec/gitlab/cli_spec.rb

```diff
@@ -59,6 +60,21 @@
       end
     end
 
+
+    context "when command with json output" do
+      before do
+        stub_get("/user", "user")
+        args = ['user', '--json']
+        @output = capture_output { Gitlab::CLI.start(args) }
+      end
+
+      it "should render output as json" do
+        expect(JSON.parse(@output)['result']).to eq(JSON.parse(File.read(File.dirname(__FILE__) + '/../fixtures/user.json')))
```

Comment:
Hi @asedge - I investigated that and from my understanding this is what happens in the test. A User is faked via the stub and we check that the JSON formatted output has the expected fields. So the attributes like email etc. are checked "implicitly".